pkg/config/config_parser.go

```diff
@@ -236,3 +241,72 @@ func decodeCloudQueryBlock(block *hcl.Block, ctx *hcl.EvalContext) (CloudQuery,
 	cq.Logger = &logging.GlobalConfig
 	return cq, gohcl.DecodeBody(block.Body, ctx, &cq)
 }
+
+func validateConnection(connection *Connection) diag.Diagnostics {
```

Comment:
document this function

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/913/commits/67837b2ad64d2cadb4358963cd78c6414a4fceb7